style.css

```diff
@@ -108,26 +107,544 @@ a {
+  line-height: 24px;
+  font-weight: 500;
+  color: #344563;
+  margin: 10px 0;
+}
+
+.block-1 .tags,
+.block-2 .tags {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 16px;
+  margin: 14px 0 20px 0;
+}
+
+.tags li {
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+}
+
+.actions {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.btn {
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: #fff;
+}
+
+.btn:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+}
+
+.btn:active {
+  background: #2230d2;
+}
+
+/* about */
+.about {
+  background-color: white;
+  border-top-right-radius: 80px;
+  padding-bottom: 80px;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 100px 20px 20px 20px;
+}
+
+.text-width {
+  width: 40px;
+}
+
+.social-media-2 {
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin: 10px 0;
   width: 200px;
   height: 40px;
 }
+
+.skill-list {
+  padding: 20px 20px 100px 20px;
+}
+
+.languages {
+  padding: 10px 10px;
+}
+
+.language-list {
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  justify-content: space-around;
+  gap: 10px;
+  padding: 20px 10px;
+}
+
+.language-list li {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 10px;
+  background: #f7f7f9;
+  border-radius: 2px;
+  padding: 5px;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.lg2 {
+  display: none;
+}
+
+.frameworks {
+  padding: 10px 10px;
+}
+
+hr {
+  height: 1px;
+  border: none;
+  background: #dfe1e6;
+}
+
+.frameworks,
+.languages,
+.skills {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.skills {
+  padding: 10px 10px;
+}
+
+/* contact me */
+.contact-me {
+  background: #6070ff;
+  border-top-left-radius: 80px;
+  padding-bottom: 20px;
+}
+
+.bg-color-1 {
+  background-color: #fff;
+}
+
+.contact-container {
+  padding: 100px 0 0 0;
+}
+
+.lg-text-4 {
+  color: #fff;
+  font-size: 40px;
+  font-weight: bolder;
+  line-height: 48px;
+  letter-spacing: 0.37px;
+  margin: 0 20px 10px 20px;
+  text-align: center;
+}
+
+.image-container {
+  display: flex;
+  align-items: self-start;
+  justify-content: right;
+}
+
+.bg-image-2 {
+  background-image: url("./assets/contact-form-shapes-mobile.svg");
+  background-position: 80px 5px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.sm-text-3 {
+  font-weight: 300;
+  font-size: 24px;
+  text-align: center;
+  color: #ebebff;
+  margin: 0 20px 40px 20px;
+}
+
+/* form */
+.form {
+  display: flex;
+  margin: 20px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
```

Comment:
- [ ] Kindly make sure that your `Get in touch` form button is aligned in the center like in the Figma design. :wink: 

     - ### Your design (width: 787px)
     ![image](https://user-images.githubusercontent.com/87197412/186939026-4aa971e7-508f-4443-bc60-d10bd9fd27b1.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186939147-86dccc29-b9bb-444b-bd57-085558aeb415.png)

Reply:
Resolved